components/01-atoms/text/text/07-byline.twig

```diff
@@ -0,0 +1,35 @@
+{#
+/**
+ * Available variables:
+ * - byline_base_class - the base classname
+ * - byline_modifiers - array of modifiers to add to the base classname
+ * - byline_blockname - blockname prepended to the base classname
+ * - byline_content - the content of the byline (typically text)
+ * - byline_author - the author of the byline (typically text)
+ * - byline_pre - prefaces the author of the byline (typically text)
+ *
+ * Available blocks:
+ * - byline_content - used to replace the content of the blockquote with something other than text
+ *   for example: A formatted text field in Drupal
+ */
+#}
+
+{% set byline__base_class = byline__base_class|default('byline') %}
+
+{# {% set date_time__attributes = {
+  datetime: date_time__start|date(format__iso),
+  class: bem('date-time'),
+} %} #}
+
+
+
+  <byline {{ add_attributes(byline__attributes) }}>
```

Comment:
I think it would be worth researching this element a little more and figure out if this is the most semantic way to markup author type content. `<byline>` seems like it was an HTML spec back in HTML2 but doing a quick google search on `<byline>` is coming up with much.